$OpenBSD: patch-rsa-sign-tr_c,v 1.1 2018/12/11 10:46:06 jca Exp $

Don't use C99 construct:
  rsa-sign-tr.c:243: error: 'for' loop initial declaration used outside C99 mode

Index: rsa-sign-tr.c
--- rsa-sign-tr.c.orig
+++ rsa-sign-tr.c
@@ -239,8 +239,9 @@ static int
 sec_equal(const mp_limb_t *a, const mp_limb_t *b, size_t limbs)
 {
   volatile mp_limb_t z = 0;
+  size_t i;
 
-  for (size_t i = 0; i < limbs; i++)
+  for (i = 0; i < limbs; i++)
     {
       z |= (a[i] ^ b[i]);
     }
